src/Fetch.res

```diff
@@ -82,6 +83,20 @@ module Headers = {
   external get: (t, string) => option<string> = "get"
   @send external has: (t, string) => bool = "has"
   @send external set: (t, string, string) => unit = "set"
+
```

Comment:
Could you add a comment here to note that the below is from the `iterable` protocol?